terraform/eks-cloudwatch/app_mesh.tf

```diff
@@ -0,0 +1,156 @@
+# ------------------------------------------------------------------------
```

Comment:
may i know the reason why we don't do the logic in `eks` folder? the folder name `eks-cloudwatch` seems not verbose enough? hard to guess its usage

Reply:
* EKS seems to have its own testcases. The current folder `eks` is too general yet serves for a very specific testing purpose. For instance, it requests user to set up cortex in the account, which is of no interest when testing container insight features. Vice versa, the framework should not ask users to install AppMesh when testing EKS merely.
* The new `eks-cloudwatch` uses a separate container insight configuration rather than the default one, which conflicts with the one that eks should use. Related PR: https://github.com/aws-observability/aws-otel-collector/blob/92414f1de2fba3a9c846922304b4bce81252a952/config/eks/prometheus/config-all.yaml

So I don't feel it's a good idea to couple eks and the test cases I'm adding. Perhaps `eks-containerinsight` is more self-explained? btw The name should be able to split `ecs` and `eks` cuz we would have container insight test cases on ECS soon as well.

Reply:
I think `containerinsight-eks-prometheus` should be good and I can do `containerinsight-ecs-prometheus`, cause they are just setting up container insight auto dashboard specific workloads. btw: there is little to share between eks and ecs containersight when setting up environment so we don't need a common module for containerinsight.

Reply:
@wyTrivail Let me know if any of the proposed names sounds good to you.

Reply:
containerinsight-eks-prometheus sounds good